pacemaker_metrics.go

```diff
@@ -93,7 +98,8 @@ var (
 		"stonith_enabled":     NewMetricDesc("pacemaker", "stonith_enabled", "Whether or not stonith is enabled", nil),
 		"fail_count":          NewMetricDesc("pacemaker", "fail_count", "The Fail count number per node and resource id", []string{"node", "resource"}),
 		"migration_threshold": NewMetricDesc("pacemaker", "migration_threshold", "The migration_threshold number per node and resource id", []string{"node", "resource"}),
-		"config_last_change":  NewMetricDesc("pacemaker", "config_last_change", "Indicate if a configuration of resource has changed in cluster", []string{}),
+		"config_last_change":  NewMetricDesc("pacemaker", "config_last_change", "Indicate if a configuration of resource has changed in cluster", nil),
+		"constraints":         NewMetricDesc("pacemaker", "constraints", "Indicate if a constraints of specific type is present per resource ID", []string{"type", "id"}),
```

Comment:
I think the help here is a little bit misleading: the `id` label is actually the id of the constraint, not of the resource.
I know that the ban id usually also contains the resource id, but after all, it's an opaque value so I think we should add the `resource` label anyway, and distinguish the two so that users can join `contraints` and `resources` metrics.